app/assets/javascripts/utils/course_utils.coffee

```diff
@@ -17,8 +20,12 @@ class CourseUtils
 
   # This builds i18n interface strings that vary based on state/props.
   i18n: (message_key, prefix, default_prefix = 'courses') ->
-    key_prefix = prefix
-    key_prefix ||= default_prefix
-    I18n.t("#{key_prefix}.#{message_key}")
+    if prefix
+      message = I18n.t("#{prefix}.#{message_key}")
+
+    if !message || message.startsWith("[missing")
```

Comment:
@ragesoss thanks! Mmm so, yes looks like it is correct, my bad:
https://github.com/fnando/i18n-js
(A comment somewhere that this depends on leaving on the default behaviour for missing messages might be useful, in any case? I don't see any other error flag for missing messages provided by the library... Seems ironic that an i18n library makes you check for a literal English string, though... )